src/python/genny/genny_auto_tasks.py

```diff
@@ -0,0 +1,74 @@
+import os
```

Comment:
let's add a module-level comment at the top here that says how this is used

Reply:
Done.